src/main/java/com/nakal/imageutil/ImageUtil.java

```diff
@@ -193,15 +194,9 @@ public String fetchValueFromYaml(String screenName) throws FileNotFoundException
 
     public boolean checkIfMaskRegionExists(String screenName) throws FileNotFoundException {
         getFuzzValue();
-        if ((YamlReader.getInstance().getValue(System.getenv("MASKIMAGE"))) != null) {
-            if (((LinkedHashMap) YamlReader.getInstance().getValue(System.getenv("MASKIMAGE"))).get(screenName)
-                    != null
-                    || ((LinkedHashMap) YamlReader.getInstance().getValue(System.getenv("MASKIMAGE"))).get(screenName)
-                    != null) {
-                return true;
-            }
-        }
-        return false;
+
+        Object mask=YamlReader.getInstance().getValue(maskImage);
+        return mask!=null && ((LinkedHashMap)mask).get(screenName)!=null ? true : false;
```

Comment:
No need of ternary operators here...
`return mask != null && ((LinkedHashMap)mask).get(screenName) != null` should return a boolean